purchase_order_variant_mgmt/wizard/purchase_manage_variant.py

```diff
@@ -91,16 +90,13 @@ def button_transfer_to_order(self):
                 order_line._onchange_quantity()
                 order_line_vals = order_line._convert_to_write(
                     order_line._cache)
-                purchase_order.order_line.create(order_line_vals)
+                purchase_order.order_line.browse().create(order_line_vals)
```

Comment:
Is the `browse` here really needed ? IMO would look cleaner to use `self.env['purchase.order.line']`

Reply:
I did the change because in the sale_order_variant_mgmt (already migrated) appears that way. I don't know the reason and i copy it. It is with the self.env['purchase.order.line'] now.

Reply:
It's needed, yes, because you will have a singleton error if not when having existing lines. It's a curious side effect here in v11.

Reply:
Ahhh, OK i underestand the empty browse now, but with self.env['purchase.order.line'].create() we avoid any singleton error because we don't need to acces to purchase_order.order_line instance to create. I think that using self.env[] is the normal way to create a record.

Reply:
Yeah, that's another option indeed. Here we make the browse for not having to enumerate the exact model, as we have it in the field, but if you see it more readable, please change it.